src/Oracles/Config/Setting.hs

```diff
@@ -173,3 +188,15 @@ cmdLineLengthLimit = do
         (False, True) -> 200000
         -- On all other systems, we try this:
         _             -> 4194304 -- Cabal library needs a bit more than 2MB!
+
+-- mk/install.mk: 108
+-- Logics of Autoconf, which might depend on the platform
+bindir, libdir, ghclibdir :: Action String
```

Comment:
Same comment as above: these settings could also be coming from `cfg/system.config.in`.